### PR TITLE
Django3.2 upgrade

### PR DESCRIPTION
**Pull request by UsamaSadiq**
## Description
- Upgraded requirements with `Django 3.2`.
- Bumped the version to `3.2.0`.
- Removing package `six` from requirements causes test failures so that'll be handled in another PR.

### CODE REVIEW

File: requirements/constraints.txt [hunk 1]
```diff
@@ -1,5 +1,2 @@
-
 # Django LTS version
-django>=2.2,<2.3
-
-futures==3.1.1; python_version > "2.7"
+django>=3.2,<3.3
```
Comment (mraarif) [inline requirements/constraints.txt:2]: shouldn't this be `Django<4.0`

Comment (UsamaSadiq) [reply requirements/constraints.txt:2]: I used `django<3.3` to keep it similar to what we have in [common constraints](https://github.com/edx/edx-lint/blob/master/edx_lint/files/common_constraints.txt#L16).

Comment (mraarif) [reply requirements/constraints.txt:2]: there's no Django 3.3, that's why it should be `<4.0`

Comment (UsamaSadiq) [reply requirements/constraints.txt]: Updated the constraint.

**Review by iamsobanjaved — APPROVED** [review]

**Review by aht007 — APPROVED** [review]

**Pull request merged**